docs/features/wait/exec.md

```diff
@@ -0,0 +1,19 @@
+# Exec Wait strategy
```

Comment:
```suggestion
# Exec Wait Strategy
```
After looking through the whole PR, I think we should capitalize `Strategy` in every heading, WDYT?